src/features/dialog-manager/reducer.js

```diff
@@ -9,12 +9,6 @@ const initialState = {
     gameOver: false,
     gameStart: true,
     gameInstructions: false,
-    characterCreation: false,
```

Comment:
Why is this moved from this section?  I feel that this is an unnecessary move.  
The reason it was there in the first place was for consistency; everywhere you will find characterCreation, it will always be straight after gameInstructions (which I did on purpose) so that it's easier to find and is consistent.  For those reasons, I think moving them to the bottom is unnecessary.

Reply:
It was moved so as to be consistent with the other additions that were made. It seemed unclean to have an 'Object block' in the middle of the list, as opposed to being grouped with the other 'Object blocks' (like `abilities` and `ability_minimum`). 
As far as ordering it after `gameInstructions`, I'm afraid I can't see the logic in that decision, as the `characterCreation` dialog comes before `gameInstructions` in the menu flow.